### PR TITLE
WIP: VZ 4207: OpenSearch Support (Console)

### PR DESCRIPTION
**Pull request by anders-swanson**

### CODE REVIEW

File: integtest/config.uitest.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "prometheus": {
     "url": ""
   },
-  "kiali": {
+  "osd": {
```
Comment (desagar) [inline integtest/config.uitest.json:28]: don't you want to replace kibana with osd, rather than kiali?

Comment (anders-swanson) [reply integtest/config.uitest.json:28]: I definitely do, thank you.

File: integtest/scripts/edit_integ_test_config.sh [hunk 2]
```diff
@@ -9,18 +9,18 @@ INPUT_CONFIG_FILE=$1
 
 CONSOLE_HOST="$(kubectl get ingress verrazzano-ingress -n verrazzano-system -o jsonpath='{.spec.rules[0].host}')"
 GRAFANA_HOST="$(kubectl get ingress vmi-system-grafana -n verrazzano-system -o jsonpath='{.spec.rules[0].host}')"
-KIBANA_HOST="$(kubectl get ingress vmi-system-kibana -n verrazzano-system -o jsonpath='{.spec.rules[0].host}')"
+OSD_HOST="$(kubectl get ingress vmi-system-kibana -n verrazzano-system -o jsonpath='{.spec.rules[0].host}')"
```
Comment (desagar) [inline integtest/scripts/edit_integ_test_config.sh:12]: is there going to be a follow up PR to change the deployment name to vmi-system-osd or some such?

**Review by desagar — APPROVED** [review]

File: integtest/scripts/edit_integ_test_config.sh (hunk 2 above)
Comment (anders-swanson) [reply integtest/scripts/edit_integ_test_config.sh:12]: Yes, there's a separate story for cleanup of all these references (objects, code, docs, and others).

**Pull request merged**